src/resources/postcss/tickets-tables.pcss

```diff
@@ -244,49 +173,44 @@
 		}
 	}
 
+	@media screen and ( max-width: 782px ) {
+		td.ticket_edit {
+			position: absolute;
+			top: 0;
+			right: 0;
+			width: 37px;
+		}
 
-	button.tribe-toggle-row {
-		background: rgba(0, 0, 0, 0);
-		border: none;
-		display: none;
-		height: 40px;
-		outline: none;
-		padding: 0;
-		position: absolute;
-		right: 8px;
-		top: 10px;
-		width: 40px;
+		tr {
+			position: relative;
+			border-bottom: 1px solid var(--border-color);
+		}
 
-		@media screen and (max-width: 782px) {
-			display: block;
+		th.column-primary ~ th {
```

Comment:
This could also be done with `th:not( .column-primary )` but it works the same 😉